src/components/List/Pagination.tsx

```diff
@@ -1,16 +1,18 @@
 import React from 'react'
 import { Pagination, Dropdown, Grid } from 'semantic-ui-react'
 import { useRouter } from '../../utils/hooks/useRouter'
-import strings from '../../utils/constants'
+import { LanguageStrings } from '../../contexts/Localisation'
```

Comment:
This is just a Type, so I can refer to the type in the Pagination props.

For this component, it was easier to pass the strings down as props rather than import. That's because I needed to refer to a string in the props themselves as a default, which wasn't possible if the `useLanguageProvider()` hadn't been called yet.

```js
const PaginationBar: React.FC<PaginationProps> = ({
  totalCount,
  strings,
  perPageText = strings.LABEL_LIST_PER_PAGE,
}) => {
  const { query, updateQuery } = useRouter()...
\\ ... Hook would be here, but it's too late!
```